FirstProject/BasedCalculator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FirstProject
+{
+    internal class BasedCalculator
+    {
+
+        public int Addiction(int x, int y)
+        {
+            return x + y;
+        }
+        public int Substraction(int x, int y)
```

Comment:
Raczej jestem przeciwny dawaniu wartości x i y, bo może się zdarzyć ze nowa osoba nie bedzie wiedziała o co chodzi po nazwie, wolę np value1, value2, albo number1, number2 czy cos w tym stylu - coś aby bardziej wskazywało na to że to jest wartość liczbowa na której działamy

Reply:
I to tez sie tyczy każdej metody